js/widgets/status.js

```diff
@@ -330,4 +332,4 @@ class StatusMatrix {
 
         logo.updatingStatusMatrix = false;
     }
-}
+}
```

Comment:
always add new line at eof
```suggestion
}

```

Reply:
Yeah, sure.
Thanks.